pkg/cc/config/config.go

```diff
@@ -1,39 +1,52 @@
 package config
 
 import (
+	"fmt"
 	"io"
+	"os"
+	"path/filepath"
 	"strings"
 
 	"github.com/aserto-dev/aserto/pkg/auth0"
 	decisionlogger "github.com/aserto-dev/aserto/pkg/decision_logger"
+	"github.com/aserto-dev/aserto/pkg/filex"
 	"github.com/aserto-dev/aserto/pkg/x"
 	"github.com/mitchellh/mapstructure"
 	"github.com/pkg/errors"
 	"github.com/spf13/viper"
 )
 
-var (
-	EnvironmentErr = errors.New("unknown environment")
-)
+const ConfigPath = "config.yaml"
 
 // Overrider is a func that mutates configuration.
 type Overrider func(*Config)
 
 type Auth struct {
-	Issuer   string `json:"issuer"`
-	ClientID string `json:"client_id"`
-	Audience string `json:"audience"`
+	Issuer   string `json:"issuer" yaml:"issuer"`
+	ClientID string `json:"client_id" yaml:"client_id"`
+	Audience string `json:"audience" yaml:"audience"`
 }
 
 func (auth *Auth) GetSettings() *auth0.Settings {
 	return auth0.GetSettings(auth.Issuer, auth.ClientID, auth.Audience)
 }
 
 type Config struct {
-	TenantID       string                `json:"tenant_id"`
-	Services       x.Services            `json:"services"`
-	Auth           *Auth                 `json:"auth"`
-	DecisionLogger decisionlogger.Config `json:"decision_logger"`
+	Context        Context               `json:"context" yaml:"context"`
+	Services       x.Services            `json:"services" yaml:"services"`
+	Auth           *Auth                 `json:"auth" yaml:"auth"`
+	DecisionLogger decisionlogger.Config `json:"decision_logger" yaml:"decision_logger"`
```

Comment:
I don't see `DecisionLogger` being used anywhere. It's being set on `CommonCtx` but no one reads it.